integrations/woopra/lib/index.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+var each = require('component-each');
+var integration = require('@segment/analytics.js-integration');
+var toSnakeCase = require('to-snake-case');
+var is = require('is');
+var foldl = require('@ndhoule/foldl');
+var json = require('json3');
+var isostring = require('isostring');
+var time = require('unix-time');
+
+/**
+ * Expose `Woopra` integration.
+ */
+
+var Woopra = module.exports = integration('Woopra')
+  .global('woopra')
+  .option('domain', '')
+  .option('cookieName', 'wooTracker')
+  .option('cookieDomain', null)
+  .option('cookiePath', '/')
+  .option('ping', true)
+  .option('pingInterval', 12000)
+  .option('idleTimeout', 300000)
+  .option('downloadTracking', true)
+  .option('outgoingTracking', true)
+  .option('outgoingIgnoreSubdomain', true)
+  .option('downloadPause', 200)
+  .option('outgoingPause', 400)
+  .option('ignoreQueryUrl', true)
+  .option('hideCampaign', false)
+  .tag('<script src="//static.woopra.com/js/w.js">');
+
+/**
+ * Initialize.
+ *
+ * http://www.woopra.com/docs/setup/javascript-tracking/
+ */
+
+Woopra.prototype.initialize = function() {
+  /* eslint-disable */
+  (function(){var i, s, z, w = window, d = document, a = arguments, q = 'script', f = ['config', 'track', 'identify', 'visit', 'push', 'call'], c = function(){var i, self = this; self._e = []; for (i = 0; i < f.length; i++){(function(f){self[f] = function(){self._e.push([f].concat(Array.prototype.slice.call(arguments, 0))); return self; }; })(f[i]); } }; w._w = w._w || {}; for (i = 0; i < a.length; i++){ w._w[a[i]] = w[a[i]] = w[a[i]] || new c(); } })('woopra');
```

Comment:
I guess this is supposed to be normal but if we could have a comment about the where it come from and why.

Reply:
Are you referring to the snippet? or the integration itself? If it's the later, we have it :) Check the [commit](https://github.com/segmentio/analytics.js-integrations/pull/106/commits/a724dd909f7d2da8a76d4efb94a81efe00a2feaf) and the [original repo](https://github.com/segment-integrations/analytics.js-integration-woopra).